pkg/cidata/cidata.TEMPLATE.d/lima.env

```
@@ -54,6 +54,9 @@ LIMA_CIDATA_SKIP_DEFAULT_DEPENDENCY_RESOLUTION=
 LIMA_CIDATA_VMTYPE={{ .VMType }}
 LIMA_CIDATA_VSOCK_PORT={{ .VSockPort }}
 LIMA_CIDATA_VIRTIO_PORT={{ .VirtioPort}}
+LIMA_CIDATA_DOCKER_PORT_MONITOR_SOCKETS={{ .PortMonitor.Docker }}
+LIMA_CIDATA_CONTAINERD_PORT_MONITOR_SOCKETS={{ .PortMonitor.Containerd }}
+LIMA_CIDATA_KUBERNETES_SERVICE_WATCHER_CONFIGS={{ .PortMonitor.Kubernetes }}
```

These names are getting kind of long, I wonder if we should drop the `_SOCKET` and `_CONFIGS` suffixes since the Go struct fields don't have them either.

I think the shared `PORT_MONITOR_` prefix should come first (and should also be used for Kubernetes), so maybe:

```suggestion
LIMA_CIDATA_PORT_MONITOR_DOCKER={{ .PortMonitor.Docker }}
LIMA_CIDATA_PORT_MONITOR_CONTAINERD={{ .PortMonitor.Containerd }}
LIMA_CIDATA_PORT_MONITOR_KUBERNETES={{ .PortMonitor.Kubernetes }}
```

Finally, these variables need to be documented in `web/content/en/docs/dev/internals.md`.